internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -385,6 +386,13 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIMessageToBedrockMes
 func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) ResponseHeaders(headers map[string]string) (
 	headerMutation *extprocv3.HeaderMutation, err error,
 ) {
+	if v, ok := headers[statusHeaderName]; ok {
+		if v, err := strconv.Atoi(v); err == nil {
+			if !isGoodStatusCode(v) {
+				return nil, nil
+			}
+		}
```

Comment:
also i am still not sure why if this is a bad status code we can continue and when good we return early. could you put the detailed code comments. This will be hard to understand by anyone without the extensive knowledge. I believe the comments will be helpful for future contributors (including me!)

Reply:
I rewrote the code in a way that it does not require this logic. Before it incorrectly returns a content type check error on error response. I am not a fan of writing too much code comments which means the code is not self-explainable.